recipe/meta.yaml

```
@@ -1,42 +1,69 @@
 {% set name = "python_http_client" %}
-{% set version = "3.3.2" %}
-{% set file_ext = "tar.gz" %}
-{% set hash_value = "67e6a7bea19b03e14dc971480d3531b80becfc203d6c69478561bf7844d52661" %}
+{% set version = "3.3.7" %}
 
 package:
   name: '{{ name|lower }}'
   version: '{{ version }}'
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: '{{ hash_value }}'
+  url: https://github.com/sendgrid/{{ name.replace('_', '-') }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 4b87ce2dbebae18e21453ac1b8c7ea31ef3139c94decc85ff5a07d4203671c56
 
 build:
+  skip: true  # [py<34 or py>310]
   number: 0
-  noarch: python
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
 
 requirements:
   host:
     - python
     - pip
+    - wheel
```

```suggestion
    - setuptools
    - wheel
```

https://github.com/sendgrid/python-http-client/blob/3.3.7/setup.py#L5